lib/bignum.toit

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2022 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the lib/LICENSE file.
+
+import encoding.hex
+
+BIGNUM_ADD_ ::= 0
+BIGNUM_SUB_ ::= 1
+BIGNUM_MUL_ ::= 2
+BIGNUM_DIV_ ::= 3
+BIGNUM_MOD_ ::= 4
+
+class Bignum:
+  sign_/bool := ?
+  limbs_/ByteArray := ?
+
+  constructor .sign_/bool .limbs_/ByteArray:
+
+  constructor.from_string data/string:
+    if data[0] == '-':
+      sign_ = true
+      data = data[1..]
+    else:
+      sign_ = false
+    
+    if data.size & 0x1 !=0:
+      data = "0" + data
+    
+    limbs_ = hex.decode data
+
+  int_to_arry_ i/int -> ByteArray:
+    return #[(i >>  0) & 0xff, (i >>  8) & 0xff,
+             (i >> 16) & 0xff, (i >> 24) & 0xff]
```

Comment:
Should be called `array` or `byte_array`, not `arry`.

This fails for ints that are outside the 32 bit range.  Toit has 64 bit signed integers.

Also it uses little-endian order, but it appears that MbedTLS uses big-endian order for import and export.  So it's unclear to me how this works.

I suggest `import binary show BIG_ENDIAN` and then:

```suggestion
  int_to_arry_ i/int -> ByteArray:
    result := ByteArray 8
    BIG_ENDIAN.put_int64 result 0 i
    return result
```

Reply:
Although MbedTLS use big-endian format to store data in MPI buffer, but this is raw data which read by MPI APIs, so we should store it as little-endian.